### DISCUSSION TITLE
Loading animation

### DISCUSSION THREAD

**Discussion by shehan360** [clientIO/joint#2560] [Q&A] [answered]
### Introduction

Hey guys, is there any way to easily show an loading animation (ex- https://samherbert.net/svg-loaders/) on a dialog popup or something else?

I have to do an API call on a click of a button, freeze interaction, and show a loading animation till the API has returned.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
There is currently no loader in _JointJS_.
However, the implementation of a full-screen blocking loader is very easy. See, for example, [StackOverflow](https://stackoverflow.com/questions/68319720/how-to-make-my-loading-screen-div-take-up-my-whole-entire-screen-even-everythin).
